labelbox/client.py

```diff
@@ -1786,3 +1790,156 @@ def get_batch(self, project_id: str, batch_id: str) -> Entity.Batch:
             experimental=True)["project"]["batches"]["nodes"][0]
 
         return Entity.Batch(self, project_id, batch)
+
+    def send_to_annotate_from_catalog(self, destination_project_id: str,
+                                      task_queue_id: Optional[str],
+                                      batch_name: str,
+                                      data_rows: DataRowIdentifiers,
```

Comment:
Something we discussed earlier with @karenkyang that it may not be clear to the user that `UniqueIds` is `DataRowIds`. For running the foundry app, I have annotated data_rows as: `Union[DataRowIds, GlobalKeys]`. https://github.com/Labelbox/labelbox-python/pull/1296/files#diff-3755e76da3ad93f8739b50227489ef9f27db2432abd67ca08999f57293ca2794R1793

Lets decide on how we want to annotate data_rows and make it consistent.